crates/sla/src/tests.rs

```diff
@@ -140,105 +132,9 @@ fn test_event_update_relayer_sla_succeeds() {
 #[test]
 fn test_event_update_relayer_sla_limits() {
     run_test(|| {
-        Sla::get_relayer_stake.mock_safe(|_| MockResult::Return(FixedI128::from(ALICE_STAKE)));
-
         // start at 99.5, add 1, result should be 100
         <RelayerSla<Test>>::insert(ALICE, FixedI128::checked_from_rational(9950, 100).unwrap());
         Sla::event_update_relayer_sla(&ALICE, RelayerEvent::BlockSubmission).unwrap();
         assert_eq!(<RelayerSla<Test>>::get(ALICE), FixedI128::from(100));
     })
 }
-
-#[test]
-fn test_event_update_relayer_total_sla_score() {
```

Comment:
we should still have the test that updates the relayer total SLA score

Reply:
As part of the overhaul I removed the `TotalRelayerScore` since we will remove the relayer stakes (and we also had duplicate tracking in the `sla` pallet).

Reply:
@gregdhill even if we remove `TotalRelayerScore` and the relayer stake, the relayers would still have an SLA so we should have a test for it. Or is there something I'm missing here?